src/node.cc

```diff
@@ -3901,9 +3906,7 @@ Environment* CreateEnvironment(Isolate* isolate,
       HandleCleanup,
       nullptr);
 
-  if (v8_is_profiling) {
-    StartProfilerIdleNotifier(env);
-  }
+  StartProfilerIdleNotifier(env);
```

Comment:
The overhead is trivial (set one flag), and does not seem to be worth the trouble of being conditional.